source/_integrations/gree.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Gree
+description: Instructions on how to integrate Gree Smart devices within Home Assistant.
+ha_category:
+  - Climate
+ha_release: 103.0
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@cmroche'
+ha_domain: gree
+---
+
+The `gree` platform allows you to control a [Gree Smart HVAC](http://global.gree.com/).
+
+### Setup
+
+Go to the integrations page in your configuration and click on new integration -> Gree.
+
+Gree Smart compatible devices are detected and added automatically.
+
+### YAML Configuration
+
+YAML configuration is around for people that prefer YAML.
+To use this integration, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+gree:
+  hosts:
+    - host: IP_ADDRESS
+  discovery: true
```

Comment:
Only config variables that are required in the first example, you can add an advanced option above **supported models**.

Reply:
I only added it because it looked empty, I don't think advanced examples are necessary, it's pretty simple stuff.

Reply:
I think my recent changes resolved the issue, please open a new one if you disagree of course :)